cooldowns_deathknight.lua

```diff
@@ -238,9 +239,9 @@ LCT_SpellData[279302] = {
 	class = "DEATHKNIGHT",
 	specID = { SPEC_DK_FROST },
 	offensive = true,
-	talent = true,
 	duration = 10,
-	cooldown = 180

```

Comment:
if that is indeed the case, it might be more useful to use it as a baseline CD

Reply:
I guess my opinion could be biased here, I simply took the opinion from a few rank 1 or tournament level monks (from their stream, written guides, etc.), but it might not represent the whole PVP population, also early in the season and meta might shift.

I can add a comment TODO here, if the meta stays around we can make 90s baseline.